examples/ai-core/src/transcribe/groq-string.ts

```diff
@@ -7,6 +7,11 @@ async function main() {
   const result = await transcribe({
     model: groq.transcription('whisper-large-v3-turbo'),
     audio: Buffer.from(await readFile('./data/galileo.mp3')).toString('base64'),
+    providerOptions: {
+      groq: {
+        responseFormat: 'verbose_json',
+      },
+    },
```

Comment:
without that provider options, only `response.text` is set. I assume we did set the `response_format: 'verbose_json'` argument by default in v4 and it got lost in the transition